__tests__/withUAL.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import { withUAL } from '../src/components/provider/withUAL'
+
+jest.mock('../src/components/provider/UALContext', () => ({
+  UALContext: {
+    Consumer: ({ children }) => children('testContext'),
+  }
+}))
+
+describe('withUAL', () => {
+  let wrappedComponent
+  let component
+  let name
+
+  beforeEach(() => {
+    component = (props) => {
+      return <h1>Hello, {props.name}</h1>;
+    }
+    name = 'testName'
+    const WithUALComponent = withUAL(component)
```

Comment:
Personal preference, but I'd just call that `WrappedComponent`

Reply:
👍